src/bitExpert/Adroit/Responder/Responder.php

```diff
@@ -32,7 +32,7 @@
      *
      * @param Payload $payload
      * @param ResponseInterface $response
-     * @return ResponseInterface|null
+     * @return ResponseInterface
      */
     public function __invoke(Payload $payload, ResponseInterface $response);
```

Comment:
In fact I found the reason for null. This test assumes "null" is a valid return value for the __invoke() method: bitExpert\Adroit\Responder\Executor\ResponderExecutorMiddlewareUnitTest::throwsExceptionIfResponderExecutionDoesNotReturnResponse